### PR TITLE
Added AzugFR information

### PR DESCRIPTION
**Pull request by IhorLeontiev**
Adding AzugFR to the community list

### CODE REVIEW

File: 2022/azugfr/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+## Who
+
+[Want to get to know us][0]?
+
+## Links
+
+[Main website][1]  
+[Twitter][2]
```
Comment (noopman) [inline 2022/azugfr/index.md:18]: This link does not have a destination.

File: 2022/azugfr/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# AZUG
+
+![azugfr logo][7]
```
Comment (noopman) [inline 2022/azugfr/index.md:3]: This logo does not display anything!

**Review by noopman — CHANGES_REQUESTED** [review]
Looks good overall. Two small things to clear in the index.md file.

File: 2022/azugfr/index.md (hunk 1 above)
Comment (IhorLeontiev) [reply 2022/azugfr/index.md:19]: Typo from my side. Correction added

File: 2022/azugfr/index.md (hunk 2 above)
Comment (IhorLeontiev) [reply 2022/azugfr/index.md]: Typo from my side. Correction added

**Pull request merged**